book/src/support.md

```diff
@@ -16,10 +16,10 @@ considered authoritive and applies to all versions.
 
 Kanidm is released on a 3 month (quarterly) schedule.
 
-- February 1st
-- May 1st
-- August 1st
-- November 1st
+- February 1
+- May 1
+- August 1
+- November 1
```

Comment:
We could also write ``The First of November`` if that sounds/looks better.
Either way, ``1st`` isn't acceptable according to Chicago Manual, which SUSE depends on its number rules.

Reply:
`<month> <day>` reads more clearly to me, this change is OK.